app/models/pubmed_source_record.rb

```diff
@@ -66,6 +68,7 @@ def self.get_and_store_records_from_pubmed(pmids)
     end
     import source_records
   end
+  private_class_method :get_and_store_records_from_pubmed
```

Comment:
This seems to be a candidate for a private method.
```
$  git grep 'get_and_store_records_from_pubmed' # on master
app/models/pubmed_source_record.rb:    get_and_store_records_from_pubmed([pmid])
app/models/pubmed_source_record.rb:  def self.get_and_store_records_from_pubmed(pmids)
lib/tasks/cap_cutover.rake:        PubmedSourceRecord.get_and_store_records_from_pubmed(pmids_for_this_batch)
lib/tasks/cap_cutover.rake:    PubmedSourceRecord.get_and_store_records_from_pubmed(pmids_for_this_batch)
spec/models/pubmed_source_record_spec.rb:    it 'calls PubmedSourceRecord.get_and_store_records_from_pubmed' do
spec/models/pubmed_source_record_spec.rb:      expect(described_class).to receive(:get_and_store_records_from_pubmed)
```

Reply:
This PR proses to cull the cap_cutover tasks that use this public method. If they can be removed, this method could be private. It seems the wrapper in get_pubmed_source_record_for_pmid and it's call stack is a better public API that calls this method.